nvm.sh

```diff
@@ -1989,6 +1990,38 @@ nvm_get_mirror() {
   esac
 }
 
+# args: version path
+nvm_nixos_postinstall() {
```

Comment:
does this need to be a separate function?

Reply:
I chose to put it in a separate function to keep the install function cleaner